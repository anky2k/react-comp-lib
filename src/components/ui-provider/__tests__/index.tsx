import React from 'react';
import { render, cleanup } from '@testing-library/react';
import UiProvider from '../../ui-provider';

afterEach(cleanup);

test('it should render as per given styles', () => {
  const { asFragment } = render(<UiProvider />);

  expect(asFragment()).toMatchSnapshot();
});

test('it should render the theme container', () => {
  const { getByTestId } = render(<UiProvider />);
  expect(getByTestId('astra-theme-container')).toBeInTheDocument();
});
