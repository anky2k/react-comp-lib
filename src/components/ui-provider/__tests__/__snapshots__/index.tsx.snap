// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render as per given styles 1`] = `
<DocumentFragment>
  <div
    class="px-20 py-10"
    data-testid="astra-theme-container"
  />
</DocumentFragment>
`;
